package.json

```diff
@@ -48,10 +48,11 @@
     "pretest": "yarn lint",
     "test": "node ./test/runTest.js",
     "test:update": "SNAPSHOT_UPDATE=true node ./test/runTest.js",
-    "dev": "NODE_ENV=dev node ./scripts/build.js",
-    "watch": "NODE_ENV=dev,watch node ./scripts/build.js",
-    "build": "NODE_ENV=prod node ./scripts/build.js",
-    "package": "yarn build && vsce package --yarn",
+    "dev": "esbuild ./src/extension.js --bundle --platform=node --target=node16 --external:vscode --external:@typescript-eslint/parser --external:vue-eslint-parser --outfile=./dist/extension.js",
```

Comment:
My two cents, `await import()` is compiled into `require` incorrectly, you should use the hack instead of using `require`